NicholasSaravia/directory
# feature: Login

### Description

this feature gives the user the ability to login / logout of the app using supabase magic link. It also provides a route protector and layout component

### How To Test

- visit site
  - you should be redirected to login
- login via magic link  email input (only my email / jessica's email will work for this)
- go to email and click on link
  - you should be redirected to /directory
- refreshing your browser should keep you logged in
- click sign out
  - you should be redirected to /login

### Pics / Vids

![image](https://user-images.githubusercontent.com/24899931/235921378-959ef65f-3e0f-4cb2-8b1e-a55b8b35e975.png)
![image](https://user-images.githubusercontent.com/24899931/235921736-4f172993-3685-4b3c-afe8-34b16717e3af.png)

### Libraries Added

@mui/lab
@fontsource/familjen-grotesk
@supabase/supabase-js
